### PR TITLE
sphinxcontrib-{actdiag,nwdiag,seqdiag}: init at 2.0.0

### PR DESCRIPTION
**Pull request by davidtwco**
###### Motivation for this change
Add packages for `sphinxcontrib-actdiag`, `sphinxcontrib-nwdiag`, `sphinxcontrib-seqdiag`.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by davidtwco** [comment]
@SuperSandro2000 you've helpfully reviewed some of my other Python package changes, would you be able to review this?

File: pkgs/development/python-modules/sphinxcontrib-seqdiag/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+buildPythonPackage rec {
+  pname = "sphinxcontrib-seqdiag";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-THJ1ra/W2X/lQaDjGbL27VMn0lWPJApwgKMrPhL0JY0=";
+  };
+
+  buildInputs = [ mock sphinx-testing ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/sphinxcontrib-seqdiag/default.nix:20]: Are those required for tests? If they are not required at all please remove them from the setup.py.

Comment (davidtwco) [reply pkgs/development/python-modules/sphinxcontrib-seqdiag/default.nix:20]: There are tests for these packages, but I wasn't able to get `unittest` to run them after some experimentation, that appears to have been the conclusion in the existing `sphinxcontrib-blockdiag` package too:

https://github.com/NixOS/nixpkgs/blob/42add18071279a9bfbffcf807976ce72925826e1/pkgs/development/python-modules/sphinxcontrib-blockdiag/default.nix#L23-L24

I've removed these inputs and added a `pythonImportsCheck` for each package.

Comment (SuperSandro2000) [reply pkgs/development/python-modules/sphinxcontrib-seqdiag/default.nix]: Those are wrong. Test depedencies go to checkInputs.

Comment (davidtwco) [reply pkgs/development/python-modules/sphinxcontrib-seqdiag/default.nix]: They aren't there anymore, I've removed them.

**Pull request merged**